packages/core/contracts/oracle/interfaces/StakerInterface.sol

```diff
@@ -13,4 +13,10 @@ interface StakerInterface {
     function executeUnstake() external;
 
     function withdrawRewards() external returns (uint256);
+
```

Comment:
this is unrelated to the main changes of this PR but was added here to correct the incomplete interface.